mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/105 populate workspaces with desk/chair

Closes #105 

This procedurally adds a work station and a chair to all work _active_ spaces.
To verify: 
* Run the **Introduction Scenario** in Unity and in the OG
* Compare the scenes for desks and chairs (easiest to use Unity Scene view)
* Should be a desk and a chair in Moe and Curly's office, oriented as in the OG.
* Should _not_ be a desk or chair in the server room or break room (the one with the fridge). The server room gets another set of furniture not currently implemented.

* Try the **Physical Security** scenario 
* Notice a different desk (not an L-shape) for the WorkSpaces.
* Notice the 6 work stations on the East side do _not_ get desks (they are built in to the model for some reason) but they do get chairs.

Note:
* The WorkSpace furniture should be oriented correctly, but is not positioned very accurately.
* This creates a GameObject for each WorkSpace and stores their associated data object in a ScriptableVariable.
* There are other minor office items not being populated (lamps, trash cans). This is a separate issue #104.
*
![image](https://user-images.githubusercontent.com/7308321/105919607-cfe7e800-5fea-11eb-8c30-27b0cbd30065.png)
![image](https://user-images.githubusercontent.com/7308321/105922072-b34daf00-5fee-11eb-81cb-bee89f4ac37f.png)
